src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -647,6 +647,17 @@ public ResponseEntity<Mono<String>> getSubstationDiagramAndMetadata(
             new DiagramParameters(useName, centerLabel, diagonalLabel, topologicalColoring, componentLibrary), substationLayout, nodeUuid));
     }
 
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network/network-area-diagram")
```

Comment:
/studies/{studyUuid}/nodes/{nodeUuid}/network-area-diagram instead? network is useless here

Reply:
Yes, done